### PR TITLE
Update rules_java to 7.12.0 and java_tools to 13.9

### PR DESCRIPTION
**Pull request by iancha1992**

### CODE REVIEW

File: java/defs.bzl [hunk 1]
```diff
@@ -25,6 +25,8 @@ load("//java/toolchains:java_package_configuration.bzl", _java_package_configura
 load("//java/toolchains:java_runtime.bzl", _java_runtime = "java_runtime")
 load("//java/toolchains:java_toolchain.bzl", _java_toolchain = "java_toolchain")
 
+version = "7.12.0"
```
Comment (hvadehra) [inline java/defs.bzl:28]: no longer needed, please revert

**Review by hvadehra — CHANGES_REQUESTED** [review]
please revert the change to java/defs.bzl

File: java/defs.bzl (hunk 1 above)
Comment (iancha1992) [reply java/defs.bzl]: Done! Is this needed in the future? Or is it just this time only? @hvadehra

Comment (hvadehra) [reply java/defs.bzl]: Not needed ever :)

**Review by hvadehra — APPROVED** [review]

**Pull request closed without merging**